ChangeLog.d/crl-revocationDate.txt

```diff
@@ -0,0 +1,10 @@
+Security
+   * When checking X.509 CRLs, a certificate was only considered as revoked if
+     its revocationDate was in the past according to the local clock if 
```

Comment:
`tests/scripts/all.sh` points out there is trailing whitespace on lines 3, 4 and 5 of this file. Please remove it.